mac.sh

```diff
@@ -229,9 +229,8 @@ brew_install_or_upgrade 'imagemagick'
 brew_install_or_upgrade 'qt'
 brew_install_or_upgrade 'hub'
 brew_install_or_upgrade 'n'
-n 0.12 && n stable
+sudo n 0.12 && sudo n stable
```

Comment:
So apparently El Capitan has different permissions for the folders where node is installed. `sudo` is the suggestion from the `n`'s creator. After you install it with `sudo` first time all consecutive times super privileges are no longer required.